lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -1,10 +1,89 @@
 'use strict';
+var findAccessorMinMax = require('./findAccessorMinMax');
 var writeAccessor = require('./writeAccessor');
+var mergeBuffers = require('./mergeBuffers');
+var uninterleaveAndPackBuffers = require('./uninterleaveAndPackBuffers');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var WebGLConstants = Cesium.WebGLConstants;
+
 
 module.exports = cesiumGeometryToGltfPrimitive;
 
+function createAttributeSemantic(gltf, primitive, semantic, attributeLength) {
+    var type;
+    var bytesPerComponent;
+
+    switch (semantic.replace(/([0-9]|_)/g, '')) {
+        case 'POSITION':
+            type = 'VEC3';
+            bytesPerComponent = 8;
+            break;
+        case 'NORMAL':
+            type = 'VEC3';
+            bytesPerComponent = 4;
+            break;
+        case 'TEXCOORD':
+            type = 'VEC2';
+            bytesPerComponent = 4;
+    }
+
+    var componentsPerType = numberOfComponentsForType(type);
+
+    var bufferLength = attributeLength * componentsPerType * bytesPerComponent;
+    var buffer = new Buffer(bufferLength);
```

Comment:
Took me a bit to remember what my code was doing, but I do believe the buffer is eventually written to: `createAttributeSemantic` creates the buffer that `mapGeometryAttributeToPrimitive` writes to. 

Is this not actually writing to the buffer?